package.xml

```diff
@@ -57,6 +57,13 @@
   <run_depend>tf2_eigen</run_depend>
   <run_depend>tf2_geometry_msgs</run_depend>
   <run_depend>xacro</run_depend>
+  <run_depend>nodelet</run_depend>
+  <run_depend>gazebo_ros</run_depend>
+
+  <!-- The universal robot package is not released for melodic -->
+  <!-- Make sure to clone the package (melodic-devel branch) onto your workspace -->
+  <!-- <run_depend>ur_gazebo</run_time> -->
+  <!-- <run_depend>ur_description</run_time> -->
```

Comment:
Why did you comment out the `run_depend`s?
If the packages are in the same workspace, these should be fine, right?
And if they aren't then rosdep tells you exactly the same as the comments above :smiley:

Reply:
Adding those dependencies fails CI

Reply:
This has to be resolved either way, but I'll merge this now and create a corresponding issue.